org.eclipse.gef/src/org/eclipse/gef/ui/actions/SelectionAction.java

```diff
@@ -72,7 +72,8 @@ public void dispose() {
 	 *
 	 * @return A List containing the currently selected objects.
 	 */
-	protected List<?> getSelectedObjects() {
+	@SuppressWarnings("unchecked")
+	protected List<Object> getSelectedObjects() {
```

Comment:
The main motivation was to resolve a compilation error in ``getSelectedEditParts()` because of a cast from List<Object> to List<EditPart>. If it still compiles, it's fine to go back to this.

Reply:
Thanks for the quick turnaround!  🥇

Reply:
💞